src/util.cc

```diff
@@ -48,8 +48,7 @@ TwoByteValue::TwoByteValue(Isolate* isolate, Local<Value> value) {
   const size_t storage = string->Length() + 1;
   AllocateSufficientStorage(storage);
 
-  const int flags =
-      String::NO_NULL_TERMINATION | String::REPLACE_INVALID_UTF8;
+  const int flags = String::NO_NULL_TERMINATION;
```

Comment:
Indeed, this is not an acceptable change.  You could turn the flags into an optional argument that defaults to what is now or make it a template trait.